aws-frauddetector-detector/src/aws_frauddetector_detector/helpers/create_worker_helpers.py

```diff
@@ -17,6 +17,7 @@ def validate_dependencies_for_detector_create(afd_client, model: models.Resource
     _validate_event_type_for_detector_create(afd_client, model)
     _validate_rules_for_detector_create(afd_client, model)
     validation_helpers.validate_external_models_for_detector_model(afd_client, model)
+    _validate_model_versions_for_detector_create(afd_client, model)
```

Comment:
I think you might find that you want the logic in `_validate_model_versions_for_detector_create` for the update worker helpers as well. This is exactly what I did before implementing update and realizing I needed to do the same exact thing, so I moved the method into `validation_helpers`.

Reply:
+1. Relocating to `validation_helpers `